package.json

```diff
@@ -15,9 +15,9 @@
     "packages/identity-agent"
   ],
   "scripts": {
-    "clean": "npx npkill -d $(pwd) -t node_modules && npx npkill -d $(pwd)/packages -t dist",
-    "build": "npm run build --ws",
-    "dwn-server":"node node_modules/@web5/dwn-server/dist/esm/src/main.js || true"
+    "clean": "pnpm npkill -d $(pwd) -t node_modules && pnpm npkill -d $(pwd)/packages -t dist",
```

Comment:
You'll want to reverse this. First kill dist then `node_modules`. Running it as will fail to run the 2nd part bc `npkill` is deleted when rm'ing `node_modules`. Same problem exists in `tbdex-js` atm

Reply:
good catch!

just noticed that when testing locally since I hadn't yet installed `npkill` as a global package for PNPM.

Fixed in [a75f451](https://github.com/TBD54566975/web5-js/pull/419/commits/a75f4513bb6bd8865dd69be940658b556fe20f0d)